src/pfe/iterative-dev/idc-java/IDC/src/org/eclipse/codewind/iterdev/tasks/ContainerRemoveTask.java

```diff
@@ -85,37 +85,14 @@ public static boolean execute(IDCContext context) throws Exception {
 				
 				String imageName = appDB.get(Constants.DB_CONTAINER_NAME);
 				if (imageName != null) {
-					ProcessRunner pr = TaskUtils.runCmd(imageCommand + " rmi " + imageName + " -f", context,
+					ProcessRunner pr = TaskUtils.runCmd(imageCommand + " rmi -f " + imageName, context,
 							false);
 					if(pr.getErrorCode().orElse(0) != 0) {
 						Logger.error("Error code: " + pr.getErrorCode() + ", Failed to remove the image " + imageName);
 						return false;
 					} else {
 						Logger.info("Successfully removed the application image " + imageName);
 					}
-
-					String deploymentRegistry = appDB.get(Constants.DB_DEPLOYMENT_REGISTRY);
```

Comment:
How come we're removing the other image deletion commands?

Reply:
because 

1. when do buildah push it only creates one image `localhost/cw-lib222-c48bd7f0-000f-11ea-a01f `
2. we dont have sufficient RBACs to do `kubectl delete image` and it was redundant. `buildah rmi -f` handles everything

See https://github.com/eclipse/codewind/issues/991#issuecomment-550027103

Reply:
```
[root@codewind-workspacei0qma8vvth5ebpcl-68b9cf59cd-998sm /]# kubectl delete image cw-testenabledisable-0553ba10-0023-11ea-83e4
Error from server (Forbidden): images.image.openshift.io "cw-testenabledisable-0553ba10-0023-11ea-83e4" is forbidden: User "system:serviceaccount:che:che-workspace" cannot delete images.image.openshift.io at the cluster scope: no RBAC policy matched
```

Reply:
After chatting with Maysun, looks like the other project types don't do kubectl delete so this brings microprofile in line with the rest of the project types but we should investigate whether this leaves behind images on the cluster.